oshi-core/src/test/java/oshi/util/TupleTest.java

```diff
@@ -47,23 +47,22 @@ public void testTuples() {
         Quintet<String, Integer, Long, Character, BigInteger> quintet = new Quintet<>("D", 4, Long.valueOf("0"), 'd',
                 BigInteger.ZERO);
 
-        assertEquals("A", pair.getA());
-        assertEquals("B", triplet.getA());
-        assertEquals("C", quartet.getA());
-        assertEquals("D", quintet.getA());
+        assertEquals(pair.getA()+" should be equal to A","A", pair.getA());
+        assertEquals(triplet.getA()+" should be equal to B" ,"B", triplet.getA());
+        assertEquals(quartet.getA()+" should be equal to C","C", quartet.getA());
+        assertEquals(quintet.getA()+" should be equal to D","D", quintet.getA());
 
-        assertEquals(1, pair.getB().intValue());
-        assertEquals(2, triplet.getB().intValue());
-        assertEquals(3, quartet.getB().intValue());
-        assertEquals(4, quintet.getB().intValue());
+        assertEquals(pair.getB().intValue()+" should be equal to 1",1, pair.getB().intValue());
+        assertEquals(triplet.getB().intValue()+" should be equal to 2",2, triplet.getB().intValue());
+        assertEquals(quartet.getB().intValue()+" should be equal to 3",3, quartet.getB().intValue());
+        assertEquals(quintet.getB().intValue()+" should be equal to 4",4, quintet.getB().intValue());
+        assertEquals(triplet.getC().longValue()+" should be equal to "+Long.MAX_VALUE,Long.MAX_VALUE, triplet.getC().longValue());
```

Comment:
Rather than outputting the large number that max value actually is, just say "Long.MAX_VALUE" in plain words in the string.

Reply:
sure, I will work on the given comments.